.github/workflows/sbt-build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -e
+
+set -x
+
+if [ -z "$1" ]
+  then
+    echo "Scala version is missing. Please enter the Scala version."
+    echo "sbt-build-simple.sh 2.11.12"
+    exit 1
+else
+  SCALA_VERSION=$1
+  CURRENT_BRANCH_NAME="${GITHUB_REF#refs/heads/:-}"
+  if [[ "$CURRENT_BRANCH_NAME" == "master" ]]
+  then
+    sbt -J-Xmx2048m "; ++ ${SCALA_VERSION}!; clean; test; packagedArtifacts"
```

Comment:
For `master` branch, it does `packagedArtifacts` to check it does not fail in packaging doc and source.
I don't think you want to see it broken after you create a tag.